src/main/java/com/bunq/sdk/model/generated/RequestResponse.java

```diff
@@ -515,6 +534,28 @@ public void setAllowChat(Boolean allowChat) {
   }
 
   /**
+   * The credit scheme id provided by the counterparty for DIRECT_DEBIT inquiries.
+   */
+  public String getCreditSchemeIdentifier() {
+    return this.creditSchemeIdentifier;
+  }
+
+  public void setCreditSchemeIdentifier(String creditSchemeIdentifier) {
+    this.creditSchemeIdentifier = creditSchemeIdentifier;
+  }
+
+  /**
+   * The mandate id provided by the counterparty for DIRECT_DEBIT inquiries.
```

Comment:
`id` in this line and the few lines above, consistency.

Reply:
generated